src/cmd-buildextend-metal

```diff
@@ -172,7 +175,13 @@ if [ -z "${use_anaconda}" ]; then
     if [ -n "${ref_is_temp}" ]; then
         ref_arg=${commit}
     fi
-    runvm -drive "if=virtio,id=target,format=${image_format},file=${path}.tmp" -- \
+    target_drive="if=virtio,id=target,format=${image_format},file=${path}.tmp"
+    target_device=""
+    if [[ $image_format == raw && $image_type == dasd ]]; then
+        target_drive="if=none,id=target,format=${image_format},file=${path}.tmp"
+        target_device="-device virtio-blk-ccw,devno=fe.0.0004,drive=target,physical_block_size=4096,logical_block_size=4096,scsi=off"
```

Comment:
Is there a link you could throw in there with background on those values?

Reply:
ECKD DASD prefers 4096, if don't want to say 4096 only.
https://www.ibm.com/support/knowledgecenter/en/linuxonibm/com.ibm.linux.z.lgdd/lgdd_t_tsk_dasdpartit.html
https://www.kernel.org/doc/Documentation/s390/DASD

Here is the output from qemu :
```
$ qemu-system-s390x -device virtio-blk-ccw,help
...
physical_block_size=<uint16> - A power of two between 512 and 32768
logical_block_size=<uint16> - A power of two between 512 and 32768
...
```
And maybe or not related to this topic, from one of the KVM master:
https://lists.gnu.org/archive/html/qemu-devel/2010-03/msg00359.html

You prefer to put an URL in the git commit ?

Reply:
I was more confused about the "virtio-blk-ccw,devno=fe.0.0004" part, and found https://wiki.qemu.org/Documentation/Platforms/S390X. That ibm.com link is informative too though. Maybe just drop that one and "ECKD DASD prefers 4096" as a comment above this line?

Reply:
Ouch! `devno=fe.0.0004` is leaked from my testing that I once had devices conflict. Will drop it and add a comment. Be aware that `virtio-blk-ccw` is required though.